src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -283,9 +283,9 @@
         - name: --appgw-watch-namespace
           type: string
           short-summary: Specify the namespace, which AGIC should watch. This could be a single string value, or a comma-separated list of namespaces.
-        - name: --disable-sgxquotehelper
+        - name: --enable-sgxquotehelper
```

Comment:
This is a breaking change as the `--disable-sgxquotehelper` option is removed.  Please add a changelog in `HISTORY.md` marked with `[BREAKING CHANGE]`.

While breaking changes in extensions are more allowed, if you need to make the same change in `azure-cli`, please follow the [Deprecation process](https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#deprecating-commands-and-arguments) to first deprecate the  `--disable-sgxquotehelper` option and keep it for a while before removing it.

Reply:
I am trying to add a changelog in src/aks-preview/HISTORY.md. How do I know which version this release is?

For azure-cli, we have not introduced --disable-sgxquotehelper option. So this will not be a breaking change.